common/src/state/mod.rs

```diff
@@ -697,6 +697,29 @@ impl State {
                 // todo: notify user
                 log::info!("audio I/O device no longer available");
             }
+            BlinkEventKind::ParticipantMuted { peer_id } => {
+                if let Err(e) = self.ui.call_info.participant_muted(peer_id) {
+                    log::error!("{e}");
+                }
+            }
+            BlinkEventKind::ParticipantUnmuted { peer_id } => {
+                if let Err(e) = self.ui.call_info.participant_unmuted(peer_id) {
+                    log::error!("{e}");
+                }
+            }
+            BlinkEventKind::ParticipantDeafened { peer_id } => {
+                if let Err(e) = self.ui.call_info.participant_deafened(peer_id) {
+                    log::error!("{e}");
+                }
+            }
+            BlinkEventKind::ParticipantUndeafened { peer_id } => {
+                if let Err(e) = self.ui.call_info.participant_undeafened(peer_id) {
+                    log::error!("{e}");
+                }
+            }
```

Comment:
Ignore due to formatting macro allowing items that implement `Display` to be part of the string itself, with proper formatting (eg `log::error!("{e}");`).

Reply:
Logging works fine any item implementing `Display`, which also implements`ToString`. Errors have been logging as expected when doing `log::error!("{e}");` or using similar captured identifiers since rust 1.58 in 2021 edition.